src/secex-genprotimgvm-scripts/genprotimg.bu

```diff
@@ -29,6 +29,48 @@ storage:
           genprotimg -V --no-verify -i /var/genprotimg/vmlinuz -r /var/genprotimg/initrd.img  -p /var/genprotimg/parmfile -k /etc/se-hostkeys/ibm-z-hostkey-1 -o /var/genprotimg/se.img
           rm -f /var/genprotimg/signal.file
           bash /var/build/post-script.sh
+    - path: /etc/systemd/system-generators/coreos-genprotimg-generator
+      overwrite: true
+      mode: 0755
+      contents:
+        inline: |
+          #!/bin/bash
+          export PATH="/usr/bin:/usr/sbin:${PATH}"
+          set -euo pipefail
+          . /usr/lib/coreos/generator-lib.sh
+          if [ ! -z $(karg ignition.firstboot) ]; then
+              exit 0
+          fi
```

Comment:
I'm confused by this. Don't we want to run on first boot? Why are there even multiple boots at all? Isn't this just a utility VM to simulate what the genprotimg VM does?

Reply:
I generate this VM from time to time (when new hostkey arrives every ±4 months) and use it, this saves build time. But that's a good idea, to support working on firstboot as well - i'll think about it.

Reply:
Checked - doesn't work, we have `coreos-ignition-unique-boot.service`, which does its job and fails )